src/ashwini/assignment18/AsciiValueOfChar.java

```diff
@@ -0,0 +1,27 @@
+package ashwini.assignment18;
+/*
+ Program 1:  Write a method which will return ascii value of given character.
+Hint : Method signature should be int getAsciiOfChar(char ch)
+
+ */
+
+public class AsciiValueOfChar {
+	char charecter;
+	int getAsciiOfChar(char ch) {
+		charecter = ch;
```

Comment:
Why do you require to ch in charecter again ?
you can write **return ch** direct